lib/client-assets.php

```diff
@@ -55,9 +55,9 @@ function wc_admin_register_script() {
 	$content     = 'wp.i18n.setLocaleData( ' . json_encode( $locale_data ) . ', "wc-admin" );';
 	wp_add_inline_script( 'wc-components', $content, 'before' );
 
-	// Add Tracks script to the DOM if tracking is opted in.
+	// Add Tracks script to the DOM if tracking is opted in, and Jetpack is installed/activated.
 	$tracking_enabled = 'yes' === get_option( 'woocommerce_allow_tracking', 'no' );
-	if ( $tracking_enabled ) {
+	if ( $tracking_enabled && defined( 'JETPACK__VERSION' ) ) {
```

Comment:
Could we also NOT embed tracking if Jetpack is being used in development mode? https://jetpack.com/support/development-mode/